src/chrome/extension/polymer/app-missing-polymer.html

```diff
@@ -26,5 +26,5 @@
     <paper-button class='highlightedButton' raised on-tap='{{ downloadApp }}'>NEXT</paper-button>
 
   </template>
-  <script src='scripts/app-missing-polymer.js'></script>
+  <script src='app-missing-polymer.js'></script>
 </polymer-element>
```

Comment:
can we rename a file app-missing-polymer.ts -> app-missing.ts it's nested under polymer/